File: backend/src/main/java/com/backend/petplace/PetplaceApplication.java
package com.backend.petplace;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.context.properties.ConfigurationPropertiesScan;
import org.springframework.data.jpa.repository.config.EnableJpaAuditing;

@SpringBootApplication
@EnableJpaAuditing
@ConfigurationPropertiesScan
public class PetplaceApplication {

  public static void main(String[] args) {

    SpringApplication.run(PetplaceApplication.class, args);
  }
}

File: backend/src/main/java/com/backend/petplace/PetplaceApplication.kt
package com.backend.petplace

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.ConfigurationPropertiesScan
import org.springframework.data.jpa.repository.config.EnableJpaAuditing

@SpringBootApplication
@EnableJpaAuditing
@ConfigurationPropertiesScan
class PetplaceApplication {

    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            SpringApplication.run(PetplaceApplication::class.java, *args)
        }
    }
}
